twitter_api/twitterApi.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+
+import twitter
+
+class Twitter:
+
+    def __init__(self):
+        consumer_key = "WXfZoJi7i8TFmrGOK5Y7dVHon"
+        consumer_secret = "EE46ezCkgKwy8GaKOFFCuMMoZbwDprnEXjhVMn7vI7cYaTbdcA"
+        access_key = "867082422885785600-AJ0LdE8vc8uMs21VDv2jrkwkQg9PClG"
+        access_secret = "qor8vV5kGqQ7mJDeW83uKUk2E8MUGqp5biTTswoN4YEt6"
```

Comment:
Anyone could now take these credentials and use that to tweet on the account right? I think it is better to include it in a file on the server (by the owner), to be more secure, just like it is done with the github api stuff.